src/modules/hub.bicep

```diff
@@ -62,49 +65,60 @@ resource defaultTelemetry 'Microsoft.Resources/deployments@2022-09-01' = if (ena
 }
 
 // ADLSv2 storage account for staging and archive
-module storageAccount 'Microsoft.Storage/storageAccounts/deploy.bicep' = {
```

Comment:
If we're not using this module anymore, can you delete it? Same with the others.

Reply:
Removed Storage Account Module